lib/semian/lru_cache.rb

```diff
@@ -0,0 +1,54 @@
+class LRUCache
+  extend Forwardable
+  def_delegators :@table, :size, :count
+  def_delegators :@table, :empty?
+  def_delegators :@table, :keys
+  def_delegators :@table, :clear
```

Comment:
You can do all of this in one line, like in `ProtectedResource`—I think that's a bit cleaner.